components/select/Select.js

```diff
@@ -258,6 +259,18 @@ class Select extends PureComponent {
     );
   };
 
+  renderHelpText() {
+    const { helpText, inverse } = this.props;
+
+    if (helpText) {
```

Comment:
PP: don't do the check here bu in the general render function. That way only one function defines what is rendered and what not